src/tabs/ProjectsTab/MerkleTree.py.txt

```diff
@@ -0,0 +1,45 @@
+from base64 import encode

```

Comment:
Actually I don't think you need to make this file ".txt", you should try it out if you haven't already

Reply:
But also LMAO love this